iterm2-colors-solarized/README.md

```diff
@@ -14,13 +14,13 @@ from: <http://code.google.com/p/iterm2/downloads/list>
 
 Open iTerm 2, open Preferences, click on the "Profiles" (formerly Addresses,
 formerly Bookmarks) icon in the preferences toolbar, then select the "colors"
-tab. Click on the "load presets" and select "import...". Select the Solarized
+tab. Click on the "Color Presets" and select "import...". Select the Solarized
```

Comment:
Was it "load presets" in earlier versions of iTerm 2? I don't have a Mac but a web search suggests it. If so I'd rather update this to read:
> "Color Presets" ("Load Presets" in earlier versions)

Also, if you wouldn't mind, please add the missing "menu" from the above while you're there, so the sentence reads:
> Click on the [stuff we talked about above] menu and select "import...".